src/basic/glyph-util.c

```diff
@@ -41,6 +41,8 @@ const char *special_glyph_full(SpecialGlyph code, bool force_utf) {
                         [SPECIAL_GLYPH_TREE_SPACE]              = "  ",
                         [SPECIAL_GLYPH_TREE_TOP]                = ",-",
                         [SPECIAL_GLYPH_VERTICAL_DOTTED]         = ":",
+                        [SPECIAL_GLYPH_HORIZONTAL_DOTTED]       = "-",
```

Comment:
My existing systemd-sysupdated work seems to have a prettier progress bar drawing code that can be used in a bit more situations than `Prefix: <bar>`

- Defining the glyphs: https://github.com/systemd/systemd/pull/28134/commits/4fe7a0221e2a178667d27d49e172b654e72627d1
- Drawing the bar: 
https://github.com/systemd/systemd/pull/28134/commits/c40d4bc75877eee3a4f7b1b7e45b10ffe879b114#diff-67234709937e5d90da92211891f48a9afe5b27e97ecb3c8b9a42df667cc59cfeR744-R798

Reply:
Here's what that looks like, btw

![Screenshot from 2023-07-17 17-13-05](https://github.com/systemd/systemd/assets/11251606/d74fa3d6-d990-4eb9-a234-2428001b25cc)

Reply:
Oh, humpf. I missed that. I guess I really should review that PR. Sorry!

This is how the progress bar I prepped looks like:

![Screenshot from 2024-02-29 09-48-22](https://github.com/systemd/systemd/assets/2130732/11c928b5-1065-4399-ba79-331a533e6cd5)

Reply:
And you should see how that little green beam zooms across the screen. It's heart warming!